name-service/program/src/instruction.rs

```diff
@@ -89,6 +89,20 @@ pub enum NameRegistryInstruction {
     ///   2. `[writeable]` Refund account
     ///
     Delete,
+
+    /// Realloc the data of a name record.
+    ///
+    /// The space change cannot be more than `MAX_PERMITTED_DATA_LENGTH` different than
+    /// current `space`.
+    ///
+    /// Accounts expected by this instruction:
+    ///   0. `[writeable]` Name record to be reallocated
+    ///   1. `[signer]` Account owner
```

Comment:
nit: could you update this comment to the actual accounts provided? and also specify that the payer account may be refunded